app/models/user.rb

```diff
@@ -1,5 +1,15 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
+                                  foreign_key: :follower_id,
+                                  dependent: :destroy,
+                                  inverse_of: :follower
+  has_many :passive_relationships, class_name: "Relationship",
+                                   foreign_key: :followed_id,
+                                   dependent: :destroy,
+                                   inverse_of: :followed
+  has_many :following, through: :active_relationships, source: :followed
```

Comment:
has_many :through set up quan hệ many-to-many qua foreign_key ứng với tên association, nhưng trong TH này tên foreign_key khác nên phải chỉ rõ bằng source